packages/react-dev-utils/launchEditor.js

```diff
@@ -176,6 +187,17 @@ function guessEditor() {
           return [fullProcessPath];
         }
       }
+    } else if (process.platform === 'linux') {
+      const output = child_process
+        .execSync('ps --no-heading -e -o comm --sort=comm')
```

Comment:
Could we explain the significance of these flags?

Reply:
put some comment